Sources/BrowserServicesKit/Autofill/AutofillUserScript+SecureVault.swift

```diff
@@ -227,6 +228,8 @@ extension AutofillUserScript {
 
     struct RequestAvailableInputTypesResponse: Codable {
 
+        // GetAvailableInputTypesResponse: https://github.com/duckduckgo/duckduckgo-autofill/blob/main/src/deviceApiCalls/schemas/availableInputTypes.json
```

Comment:
Since the PR is still WIP, you can find the schema here: https://github.com/duckduckgo/duckduckgo-autofill/blob/7f8e8cbe54ee581f33cc82eca8c9f96135d8ef0e/src/deviceApiCalls/schemas/availableInputTypes.json.